source/devel_environ.rst

```diff
@@ -0,0 +1,153 @@
+=======================
+Development environment
+=======================
+
+.. _HashiCorps: https://www.hashicorp.com/
+.. _Vagrant: https://www.vagrantup.com/
+.. _Vagrant documentation: https://www.vagrantup.com/docs/
+.. _Wikipedia: https://en.wikipedia.org/
+.. _Vagrant 2.2.7: https://releases.hashicorp.com/vagrant/2.2.7/
+.. _VirtualBox 6.1.x: https://www.virtualbox.org/wiki/Downloads
+
+In Wikipedia_, HashiCorps_' Vagrant_ is defined as
+
+  Vagrant is an open-source software product for building and maintaining portable virtual software development environments, e.g. for VirtualBox, KVM, Hyper-V, Docker containers, VMware, and AWS. It tries to simplify the software configuration management of virtualizations in order to increase development productivity.
+
+Vagrant is used in S-ENDA for spinning up reproducible development environments.
+
+------------
+Installation
+------------
+
+S-ENDA has standardized on these versions for the development environment.
+
+* `Vagrant 2.2.7`_
+* `VirtualBox 6.1.x`_
+
+Download the software appropriate for your platform and install.
+
+-------------
+Generic usage
+-------------
+
+Below is a sample of the most used Vagrant commands. Other options for the configuration file, ``Vagrantfile``, and more advanced usage, read the `Vagrant documentation`_.
+
+* Create config file, ``Vagrantfile``, for a Ubuntu Bionic VM.
+
+  .. code-block:: bash
+
+    vagrant init ubuntu/bionic64
+
+* Start VM.
+
+  .. code-block:: bash
+
+    vagrant up
+
+* Access VM.
+
+  .. code-block:: bash
+
+    vagrant ssh
+
+* Stop VM.
+
+  .. code-block:: bash
+
+   vagrant halt
+
+* Rerun provisioning scripts.
+
+  .. code-block:: bash
+
+    vagrant provision
+
+
+--------------------
+S-ENDA configuration
+--------------------
+
+More specific Vagrant configuration for this project. This section will increase with time, with tips and extra configuration needed to run S-ENDA services.
```

Comment:
I am not sure how this is supposed to be used. Is it for local development, or is it for use on the servers? Is there more documentation that needs to be hidden for the public? Could you explain a bit more?

Reply:
Updated description.